low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -357,3 +381,190 @@ def get_nw_status(self, nw_interface, interface):
             nw_cable_conn_status = "UNKNOWN"
             # Log the error when logging class is in place.
         return nw_status, nw_cable_conn_status
+
+    def initialize_systemd(self):
+        """Initialize system bus and interface."""
+        self._bus = SystemBus()
```

Comment:
we have a generic implementation of initialize system bus and getting the interface in cortx-utils,
 Try to use that. check this file 
https://github.com/Seagate/cortx-utils/blob/main/py-utils/src/utils/service/service_handler.py
#L62   _get_systemd_interface(cls):

Reply:
Replaced `initialize_systemd` function with the one from cortx-utils.